### ISSUE TITLE
Mobileclient.get_all_songs() returns a dictionary for each track missing 'rating' key

### ISSUE THREAD

**Issue by garrett928** [simon-weber/gmusicapi#639]
I want to generate a thumbs up playlists of my music. This seems straight forward, and is documented in one of the docs, but fails due to every track missing the key 'rating'. I'm not sure if i'm missing something, or the api is broken or outdated. 
![capture](https://user-images.githubusercontent.com/22306616/53787997-78b86600-3eee-11e9-934d-36a94ea40107.PNG)

**Comment by simon-weber** [comment]
Are you sure it's every track? It's not well represented in the docs, but rating [is an optional field](https://github.com/simon-weber/gmusicapi/blob/76f49deeec7d90ee86d3dff8fc6214f620708b0e/gmusicapi/protocol/mobileclient.py#L125) -- I'd expect it to be present for anything that's been rated.

**Comment by garrett928** [comment]
Ok, I ran through my entire library and checked if a song had a rating or not. So, not every song is missing a rating. But, only 11 of the 4000+ songs return a rating. That doesn't really make sense though since I definitely have 11 whole albums thumbed up, let alone 11 songs.  I thought it might be that I only have about 11 songs personally bought or uploaded, the rest are through the subscription. But the songs that have a rating don't have any similarities and only one is one of my 11 personal songs. 

Here is the script I ran. Maybe I missed a simple step? 
```python
from gmusicapi import Mobileclient

mc = Mobileclient()
mc.oauth_login("937fd207c6af24de279b358634faa9f9914f0d47eefca8be01e2cba694c79cee")
print("Authenticated: " + str(mc.is_authenticated()))


lib = mc.get_all_songs()
count = 0

rated_songs = []

for song in lib[:500]:
    if 'rating' in song:
        print("song has rating = True")
        rated_songs.append(song)
        count += 1
    else:
        print("song has rating = False")

print("Songs with rating: " + str(count))
print("Total songs: " + str(len(lib)))

print("\nRated Songs\n")

for song in rated_songs:
    print("Song Title: " + song['title'])
    print("\tBy: " + song['artist'])


mc.logout()

```

Here is the output of the code:

![ratings](https://user-images.githubusercontent.com/22306616/53925718-f0050b80-404e-11e9-8067-631a2a5650f5.PNG)

**Comment by simon-weber** [comment]
The `for song in lib[:500]` bit will definitely skew it. I just ran this:
```python
lib = mc.get_all_songs()
print len(lib), 'total songs'

has_rating = [s for s in lib if 'rating' in s]
print 'have rating field', len(has_rating)

rated = [s for s in lib if s.get('rating') != '0']
print 'have a rating', len(rated)
```

Initially I got:
```
9471 total songs
have rating field 9471
have a rating 0
```

And then after rating a few songs, I got
```
9471 total songs
have rating field 9471
have a rating 6
```

My guess would be the discrepancy is mostly due to non-library songs, which won't be returned from get_all_songs. My library is also a non-subscription account -- maybe uploaded songs have the field by default? I also used the 5-star ratings lab back when it existed (though I've uploaded songs since).

There's some more details on the possible values over at https://github.com/simon-weber/Autoplaylists-for-Google-Music/issues/54, in case it's helpful.
